de.metas.contracts/src/main/java/de/metas/contracts/flatrate/impexp/FlatrateTermImporter.java

```diff
@@ -136,43 +143,26 @@ private void setDropShipBPartner(@NonNull final I_I_Flatrate_Term importRecord,
 		contract.setDropShip_BPartner_ID(dropShipBPartnerId);
 	}
 
-	private void setDropShipLocation(@NonNull final I_C_Flatrate_Term contract)
+	private void setDropShipUser(@NonNull final I_C_Flatrate_Term contract)
```

Comment:
it might be a good idea to make methods static, if they don't use any of the classes members..but that's only a minor tip..not a must..